docs/ldml/tr35-keyboards.md

```diff
@@ -972,14 +971,14 @@ The following elements are not imported from the source file:
 
 * * *
 
-### <a name="Element_displayMap" href="#Element_displayMap">Element: displayMap</a>
+### <a name="Element_displays" href="#Element_displays">Element: displays</a>
 
-The displayMap can be used to describe what is to be displayed on the keytops for various keys. For the most part, such explicit information is unnecessary since the `@to` element from the `keys/key` element can be used. But there are some characters, such as diacritics, that do not display well on their own and so explicit overrides for such characters can help.
+The displays can be used to describe what is to be displayed on the keytops for various keys. For the most part, such explicit information is unnecessary since the `@to` element from the `keys/key` element can be used. But there are some characters, such as diacritics, that do not display well on their own and so explicit overrides for such characters can help.
 Another useful scenario is where there are doubled diacritics, or multiple characters with spacing issues.
 
-The `displayMap` consists of a list of display subelements.
+The `displays` consists of a list of display subelements.
 
-displayMaps are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.
+displayss are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.
```

Comment:
```suggestion
`displays` elements are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.
```

The double-s may have been intentional but very confusing!

Reply:
no, it was search and replace… (displayMaps)s -> (displays)s